tests/run_bf.sh

```diff
@@ -15,4 +16,8 @@ else
 fi
 export PYTHONPATH
 
-exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
+bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json \
+        -n prplmesh_docker -x test_flows -o ${resultdir} \
+	|| exit $?
+
+exit $(jq '.tests_fail' ${resultdir}/test_results.json)
```

Comment:
if `tests_fail` has a non-numeric value, the entire script will have an error:
`exit: "non-numberic-value": numeric argument require`
because `exit` expect a number.
We might count on bft that it always be a number or we can test the value before directly transferring it to `exit`.

Reply:
I changed the script to print "Unable to parse failed test count" so that it would be easier to debug if something goes wrong with test_results.json.